src/core/server/opensearch/legacy/opensearch_client_config.ts

```diff
@@ -51,7 +51,7 @@ export type LegacyOpenSearchClientConfig = Pick<ConfigOptions, 'keepAlive' | 'lo
     | 'apiVersion'
     | 'customHeaders'
     | 'logQueries'
-    | 'requestHeadersWhitelist'
+    | 'requestHeadersAllowlist'
```

Comment:
just making sure this works right?

some of these left over non-inclusive language was actually because the [openseach-js ](https://github.com/opensearch-project/opensearch-js)client did not support the inclusive language